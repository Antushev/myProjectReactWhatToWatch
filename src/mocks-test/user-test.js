const user = {
  id: 1,
  name: `Джек`,
  email: `jack@sina.com`,
  avatar: `/wtw/static/avatar/8.jpg`
};

export {user};
