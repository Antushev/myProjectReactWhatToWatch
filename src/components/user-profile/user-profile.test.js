import React from 'react';
import renderer from 'react-test-renderer';

import {user} from './../../mocks-test/user-test.js';

import UserProfile from './user-profile.jsx';

describe(`UserProfileComponent`, () => {
  it(`UserProfileComponentSnapshot`, () => {
    const tree = renderer
      .create(
          <UserProfile
            user={user}
            authorizationStatus={`AUTH`}
            handleTypeScreenChange={() => {}}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
