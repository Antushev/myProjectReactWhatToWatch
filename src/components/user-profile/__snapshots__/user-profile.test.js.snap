// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfileComponent UserProfileComponentSnapshot 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="https://4.react.pages.academy/wtw/static/avatar/8.jpg"
      width="63"
    />
  </div>
</div>
`;
